From ab0a08cbdc8883361e233d82e2713ddc93616e46 Mon Sep 17 00:00:00 2001
From: Daniel Simon <dansimon09@gmail.com>
Date: Fri, 30 May 2025 16:37:30 -0700
Subject: [PATCH] Fix Typo in Tasking CMakeLists.txt

Fixed typo in cmake where the TBB component used by embree wasn't correctly output to cmake status.
---
 common/tasking/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/tasking/CMakeLists.txt b/common/tasking/CMakeLists.txt
index bf790ef63c..9929f3902a 100644
--- a/common/tasking/CMakeLists.txt
+++ b/common/tasking/CMakeLists.txt
@@ -16,7 +16,7 @@ ELSEIF (TASKING_TBB)
   ADD_LIBRARY(tasking STATIC taskschedulertbb.cpp)
   
   if (TARGET TBB::${EMBREE_TBB_COMPONENT})
-    message("-- TBB: reuse existing TBB::${TBB_COMPONENT} target")
+    message("-- TBB: reuse existing TBB::${EMBREE_TBB_COMPONENT} target")
     TARGET_LINK_LIBRARIES(tasking PUBLIC TBB::${EMBREE_TBB_COMPONENT})
   else()
     # Try getting TBB via config first
